### ISSUE TITLE
package requirement mismatch between cabal file and stack.yaml

### ISSUE THREAD

**Issue by flip111** [jameysharp/corrode#126]
https://github.com/jameysharp/corrode/blob/467135a0e3f1801fbe312187af649357f99efdb7/corrode.cabal#L23

https://github.com/jameysharp/corrode/blob/34547166876937efb59687bce2ee7d5d19f7d841/stack.yaml#L18

https://www.stackage.org/lts-6.6/package/language-c-0.5.0

```
» stack install
Error: While constructing the build plan, the following exceptions were encountered:

In the dependencies for corrode-0.1.0.0:
    language-c-0.5.0 must match ==0.6.* (latest applicable is 0.6.1)

Plan construction failed.
```

**Comment by flip111** [comment]
workaround: `git checkout 9b4bdd5fb9f9d1f312af63ecad918742a87677f1`

**Comment by jameysharp** [comment]
Thanks! Fixed, I think, by merging #131. Feel free to re-open if that didn't do the trick for you.
